ts/features/fims/history/store/selectors/index.ts

```diff
@@ -6,6 +7,12 @@ import { GlobalState } from "../../../../../store/reducers/types";
 export const fimsHistoryPotSelector = (state: GlobalState) =>
   state.features.fims.history.consentsList;
 
+export const isFimsHistoryLoadingSelector = (state: GlobalState) =>
+  pot.isLoading(state.features.fims.history.consentsList);
+
+export const fimsHistoryToUndefinedSelector = (state: GlobalState) =>
```

Comment:
Why not just `fimsHistorySelector`?

Reply:
since we have `fimsHistoryPotSelector`, I think that a `fimsHistorySelector` would sound redundant, so I wanted to differenciate the two